### ISSUE TITLE
[Bug]: LunarVim Info provides misleading instructions about configuring formatters

### ISSUE THREAD

**Issue by vlcinsky** [LunarVim/LunarVim#1299]
### Problem description

Steps to reproduce:
1. open e.g. python file thing.py
2. use `<space>Li` to show Lunarvim info page
3. it shows text as follows
```
  * List of supported formatters: [  ]
  * Configured formatter needs to be installed and executable.
  * Enable installed formatter(s) with following config in ~/.config/lvim/config.lua

    lvim.lang.dashboard.formatting = { { exe = '' } }
```
However, consulting https://github.com/LunarVim/LunarVim/blob/a3344203818658441d176fe279e1978505940432/lua/default-config.lua#L879-L886 the parameters for configuring should read:

```
  * List of supported formatters: [  ]
  * Configured formatters needs to be installed and executable.
  * Enable installed formatter(s) with following config in ~/.config/lvim/config.lua

    lvim.lang.dashboard.formatters = { { exe = '' } }
```

### LunarVim version

0.5.1 + (4fd72b1be2c1220a519d5b8d58219939a11df0c9)

### Neovim version (>= 0.5)

v0.5.0

### Relevant log output

```shell
logs are not relevant here, it is typo
```

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by abzcoding** [comment]
atm we don't use args for anything, hence the shorter version

**Comment by vlcinsky** [comment]
Note, that I am talking about formatters for python (not for `lvim.lang.dasbhoard`)

My local config for python is using it:
```lua
-- set a formatter if you want to override the default lsp one (if it exists)
lvim.lang.python.formatters = {
  {
    exe = "isort",
    -- all but exe is currently ignored
    -- args = {}
  },
  {
    exe = "black",
    -- all but exe is currently ignored
    -- args = {}
  }
}

lvim.lang.python.linters = {
	{
		exe = "flake8",
    -- all but exe is currently ignored
		-- args = {},
	},
}
```

And it works for me.

**Comment by vlcinsky** [comment]
[@ChristianChiarulli is using `lvim.lang.python.formatters` too](https://github.com/ChristianChiarulli/lvim/blob/master/ftplugin/python.lua) so I guess it is used.

Actually I thing, the `lvim.lang.python.formatting` key is possibly ignored so current instructions on LunarVim Info page is misleading.

**Comment by abzcoding** [comment]
when you open the `<leader>Li` in dashboard instead of the actual python file
it shows you that,
<img width="677" alt="Screen Shot 2021-08-12 at 1 34 25 PM" src="https://user-images.githubusercontent.com/10992695/129169924-be0cf165-36bd-445a-917e-52cbd959ae96.png">

**Comment by abzcoding** [comment]
my bad 🤦  you are right

**Comment by abzcoding** [comment]
gonna be fixed via #1277

**Comment by vlcinsky** [comment]
Great.
Btw the LunarVim Info page provides really great service - shows current status + provides hints how to modify that. I like it a lot.

LunarVim is becoming my fulfilled dream about ideal editor.
